.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,198 @@
+name: acceptance
+on: pull_request
+
+jobs:
+    percynonce:
+      name: Generate Percy nonce
+      runs-on: ubuntu-latest
+      steps:
+        - name: Generate Percy nonce
+          if: always()
+          run: |
+            echo ${{ github.run_id }}.$(($(date +%s))) > nonce.txt
+
+        - name: Upload nonce as artifact
+          if: always()
+          uses: actions/upload-artifact@v1
+          with:
+            name: percy
+            path: nonce.txt
+
+    acceptance:
+      needs: [percynonce]
+      runs-on: ubuntu-latest
```

Comment:
What ubuntu version is this? Is it possible to pin to xenial?

Reply:
latest is 18.04

```suggestion
      runs-on: ubuntu-16.04
```